pages/learn/develop/local-mode.md

```diff
@@ -202,13 +202,13 @@ To access the local device over [SSH][ssh], use the `{{ $names.company.short }}
 {{ $names.company.short }} ssh 192.168.86.45
 ```
 
-To connect to an application container, we can specify the service name e.g.
+To connect to an app container, we can specify the service name e.g.
 
 ```bash
 sudo {{ $names.company.short }} ssh 63ec46c.local my-service
 ```
 
-__Note:__ If an IP address or a `.local` hostname is used (instead of an application name or device UUID), `{{ $names.company.short }} ssh` establishes a direct connection to the device on port `22222` that does not rely on the {{ $names.company.short }} VPN.
+__Note:__ If an IP address or a `.local` hostname is used (instead of an app name or device UUID), `{{ $names.company.short }} ssh` establishes a direct connection to the device on port `22222` that does not rely on the {{ $names.company.short }} VPN.
```

Comment:
```suggestion
__Note:__ If an IP address or a `.local` hostname is used (instead of a fleet name or device UUID), `{{ $names.company.short }} ssh` establishes a direct connection to the device on port `22222` that does not rely on the {{ $names.company.short }} VPN.
```